qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -13,8 +13,9 @@
 import tempfile
 import traceback
 
+from . import util
+from . import coverage
 from authproxy import AuthServiceProxy, JSONRPCException
-from util import *
```

Comment:
FWIW I'd argue that the existing method of importing the `util` namespace rather than particular symbols results in less pollution, but this is a small thing.